GoogleForCreators/web-stories-wp
# Fetch page attachment link icon using API

## Summary
Fetches the icon of the Page Attachment via API and populates the field if icon found.


## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Type in `stories.google` to the Page Attachment field.
2. Verify that icon is populated automatically (it will take a bit of time to fetch it)
3. Verify that when you save the Story and see the preview, the icon displays for the Page Attachment link.
4. Now type in something incorrect
5. Verify that "Invalid link" is displayed as a warning
6. Type in `google.com`
7. Verify that the previously populated icon is not there anymore.

## Checklist




---



Fixes #9028
